doc/website/getting-started/overview.md

```diff
@@ -1,16 +1,16 @@
 # Overview
 
-This document covers the core functionality of Eclipse iceoryx and is intended to quickly get started to
-set up iceoryx applications.
+This document covers the core functionality of Eclipse iceoryx and is intended to
+provide a quick introduction to setting up iceoryx applications.
 
 ## General
 
 To set up a collection of applications using iceoryx (an _iceoryx system_), the applications need to initialize a
-runtime and create _publishers_ and _subscribers_. The publishers send data of a specific _topic_ which can be
-received by subscribers of the same topic. To enable publishers to offer their topic and subscribers to subscribe to
-these offered topics, the middleware daemon, called `RouDi`, must be running.
+runtime and create senders (_publishers_, _servers_ or _clients_) and receivers (_subscribers_, _servers_ or
+_clients_). The senders send data of a specific _topic_ which can be received by receivers of the same topic. To
+enable senders and receivers to communicate, the middleware daemon, called `RouDi`, must be running.
```

Comment:
I wouldn't mix publish-subscribe and request-response but keep them distinct. Something like
```
the applications need to initialize a runtime and create communication participants like _publisher_
and _subscriber_ or _client_ and _server_. The publisher send data of a specific _topic_ which can
be received by subscribers of the same topic. The server is waiting on a topic for requests from clients and
responds to these requests. To enable the operation of these participants, the middleware daemon,
called `RouDi`, must be running.
```
What do you think?